src/woo_publications/api/models.py

```diff
@@ -0,0 +1,69 @@
+import binascii
+import os
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from django_jsonform.models.fields import JSONField
+from phonenumber_field.modelfields import PhoneNumberField
+
+from .constants import PermissionOptions
+
+PERMISSION_SCHEMA = {
+    "type": "list",
+    "items": {
+        "type": "string",
+        "choices": [
+            {"title": label, "value": value}
+            for value, label in PermissionOptions.choices
+        ],
+        "widget": "multiselect",
+    },
+}
+
+
+class TokenAuth(models.Model):
+    token = models.CharField(_("token"), max_length=40, primary_key=True)
+    permissies = JSONField(
+        _("permission"),
+        help_text=_("The permission this API token has access to."),
+        schema=PERMISSION_SCHEMA,
+        blank=True,
+        null=True,
+    )
+    contact_persoon = models.CharField(
+        _("contact person"),
+        max_length=200,
+        help_text=_("Name of the person who can access the API"),
```

Comment:
I don't think these help texts are very useful. It's the contact person, but what they do with the API keys after that is up to them.

"Name of the person to contact about this application and the associated credentials." is probably better phrasing. The same remark applies to the other fields.